src/components/Features.svelte

```diff
@@ -0,0 +1,128 @@
+<script>
+  const features = [
+    {
+      title: "Single Sign-On",
+      description: "Login once to multiple applications",
+      imgSrc: "",
```

Comment:
Empty source makes the page look weird. Maybe use something like https://placeholder.com/ to stub the images first?

Reply:
sure, I'll do that